marketplace/release-notes/11.0.1.txt

```diff
@@ -0,0 +1 @@
+- The StringFromFile Java action now removes BOM from strings as well. (Ticket #242904)
```

Comment:
`as well` confused me. I initially thought it was already removing BOM from something but now also removes it from strings. 

Omitting `as well` may be clearer.

Reply:
Agreed, updated.